specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/azureFirewall.json

```diff
@@ -714,7 +731,8 @@
          "description":"The action type of a rule collection",
          "enum":[  
             "Allow",
-            "Deny"
+            "Deny",
+            "Alert"
```

Comment:
What does Alert mean here? Allow, but raise an alert? Does it require threat intel to be enabled?

Reply:
No, that is the regular Alert for user rules. It is not directly related to ThreatIntel, but we are enabling it just to allow user rules to function the same way as ThreatIntel in Alert mode: just log and do not take any decision to allow or deny, just continue applying the other rules.

Reply:
I didn't answer the last question... It does not require ThreatIntel at all. It can be seen as a different feature even if underneath it works the same.